service/ziti-tunnel/service/state.go

```diff
@@ -234,6 +234,9 @@ func (t *RuntimeState) CreateTun(ipv4 string, ipv4mask int, applyDns bool) (net.
 		}
 		//for windows 10+, could 'domains' be able to replace NRPT? dunno - didn't test it
 		luid.SetDNS(windows.AF_INET, []net.IP{ip}, nil)
+		cziti.SetInterfaceMetric(TunName, 5)
```

Comment:
i see i didn't add a log line in `SetInterfaceMetric` - would you log what value is being set there at debug?